src/app/pages/blog/blog.js

```diff
@@ -21,13 +27,15 @@ export function SearchResultsPage() {
     );
 }
 
-export function DefaultPage() {
+// Exported so it can be tested
+export function MainBlogPage() {
     const {pinnedStory} = useBlogContext();
-    const {search} = useLocation();
 
-    if (search) {
-        return (<SearchResultsPage />);
-    }
+    useEffect(
+        () => {document.title = 'OpenStax News';},
+        []
+    );
+
```

Comment:
I was like 99% sure that Tutor had a DocumentTitle component it rendered for this, but I can't seem to find it 😭 

Still, might be nice to extract. is probably just something like:
`const DocumentTitle = ({ title ) =>  useEffect(() => document.title = title, [])`

Reply:
Interesting to make that a component.